### ISSUE TITLE
[v4] table columns switch to mobile view too early on smaller breakpoints

### ISSUE THREAD

**Issue by jaro-io** [k-next/kirby#5388]
## description
in one of my blueprints i have a structure within a section with columns. columns are set to `1/3` and `2/3`. the structure is set to display three columns, no widths are set. the second and third column are being hidden on smaller breakpoints (< ~`1440px`).

**expected behavior**  
as you can see in the video, there is still more than enough room to show the other columns on a smaller breakpoint. so i was expecting those to switch to the mobile view _(only first column visible)_ much later.

**video**
https://github.com/getkirby/kirby/assets/20298313/245d68c1-ce97-46e4-8d79-55585255bb36

## to reproduce
```yml
columns:
    small:
        width: 1/3
        sections:
            ...
    large:
        width: 2/3
        sections:
            name:
                type: fields
                fields:
                    structure:
                        type: structure
                        columns:
                            min:
                                after: €
                            max:
                                after: €
                            amount:
                                after: €
                        fields:
                            ...
```

## your setup
**kirby version**  
[4.0.0-alpha.5](https://github.com/getkirby/kirby/releases/tag/4.0.0-alpha.5)

**Comment by afbora** [comment]
This is because it used to be based on the width of the screen, but now with v4 it is based on the container the field is in. I improved this breakpoint width by 10rem in #5394.

**Comment by jaro-io** [comment]
thank you so much for this quick response @afbora.
your commit fixes the issue for me in the example from above!
🪂 ✨ 

however, i still have many other use-cases, where the layout _(in my opinion)_ still switches to mobile too early. before upgrading to `v4` it was possible to show much more content in tables, even when there was not much space. so right now, to me it feels like this limits blueprint possibilities compared to `v3.9`. 

as you can see down below, even though the sidebar has more space now _(width in pixels)_, nearly all the columns are hidden in `v4`. i would love to be still able to style my blueprints like before.

**before:**
<img width="1728" alt="Screenshot 2023-07-20 at 00 40 59" src="https://github.com/getkirby/kirby/assets/20298313/7cb16b12-d05b-47a7-8beb-f0ee5ce4b1d6">

**after:**
<img width="1728" alt="Screenshot 2023-07-20 at 00 41 37" src="https://github.com/getkirby/kirby/assets/20298313/454aa707-1963-4981-a8ff-f2fe835f9e68">

**Comment by afbora** [comment]
I see. I've come up with three solutions;

1. Reduce the breakpoint width a bit more. Maybe 30rem.
2. Going back to media-screen width instead of Container.
3. Make breakpoint widths adjustable at blueprint level.

````yaml
# idea for option 3
fields:
  addresses:
    label: Addresses
    type: structure
    fields:
      street:
        label: Street
        type: text
      zip:
        label: ZIP
        type: text
      city:
        label: City
        type: text
    breakpoints:
      phone: 20rem
      tablet: 40rem
      desktop: 60rem
````

What do you think @distantnative ?

**Comment by distantnative** [comment]
I don't quite understand how media queries  would  solve this problem. Isn't the issue that we don't know what  content  columns will have an d thus just have to arbitrarily  pick a breakpoint?

Though I don't like now adding CSS things as blueprint options. A custom Panel CSS is already a viable solution.

**Comment by afbora** [comment]
Absolutely. Column contents can be short or long. So it would be great if it could be more adjustable by the user.

May be another example for the columns: `mobile`, `tablet`, `desktop` and widescreen.

````yaml
fields:
  structure:
    columns:
      title:
        type: text
        responsive:
          mobile: true
          tablet: true
          desktop: true
````

Or what do you think about option 3 (Reduce the breakpoint width a bit more)?

**Comment by jaro-io** [comment]
may i ask why this was closed? i mean it looks better now in alpha6 _(no weird looking tables anymore)_, but still, in my use case there is only one column shown where before it was four. i would love to see more of the table behaviour from `v3` in `v4`. but maybe that's just me? 🤷🏻‍♂️ ✨ 

<img width="1380" alt="Screenshot 2023-08-05 at 13 42 07" src="https://github.com/getkirby/kirby/assets/20298313/9cae12c2-17e9-48c4-8722-e70fc5f5c7d5">

**Comment by lukasbestle** [comment]
@jaro-io It was closed because #5394 was merged. Could you please open a new issue so it doesn't get lost? Sorry for the inconvenience.
